packages/rich-text/src/component/index.native.js

```diff
@@ -900,7 +900,9 @@ export class RichText extends Component {
 		const cssUnitOptions = { height, width, fontSize: DEFAULT_FONT_SIZE };
 		// We need to always convert to px units because the selected value
 		// could be coming from the web where it could be stored as a different unit.
-		const selectedPxValue = getPxFromCssUnit( newFontSize, cssUnitOptions );
```

Comment:
Did we experience the crash [because `getPxFromCssUnit` can return `null`](https://github.com/WordPress/gutenberg/blob/7da2bf7ee20e1b9b875f383c1fe63f20d3ae8dfb/packages/block-editor/src/utils/parse-css-unit-to-px.js#L204)?

I understand the need to add `DEFAULT_FONT_SIZE` in case an invalid value is returned from `getPxFromCssUnit`. Does this beg the question of if `null` should be the `bottom type` as opposed to something like `""`? Also, should `getPxFromCssUnit` be aware that there may be cases where a valid valid couldn't be parsed from the user input, and to allow a default value as an option to the function arguments?